### ISSUE TITLE
False positive for useless variable in try-catch

### ISSUE THREAD

**Issue by marcospassos** [slevomat/coding-standard#1470]
The following example is currently reported as a useless variable:

```php
 $response = null;

try {
    $response = $handler->handle($request);

    return $response;
} finally {
    $this->countMetric->add(1, $this->getAttributes($request, $response));
}
```

**Comment by kukulich** [comment]
What sniff does report the error? What line is reported?
